app/javascript/packages/address-search/components/full-address-search.tsx

```diff
@@ -6,6 +6,7 @@ import type { LocationQuery, FormattedLocation } from '@18f/identity-address-sea
 import FullAddressSearchInput from './full-address-search-input';
 
 function FullAddressSearch({
+  usStatesTerritories,
   registerField,
```

Comment:
I understand why you decided to pass in `usStatesTerritories` as a prop. If we use this component in sites, I imagine we will need to write an issue to get usStateTerritories in sites (help center). (I did not look into how we are getting these- jumping into another meeting).  I imagine the lift to do this in sites and pass it in is not large so am okay with passing it in- just calling it out as some additional work I think

Reply:
That's a good point. For reference how this works in identity-idp is:

1. US states and territories are [defined](https://github.com/18F/identity-idp/blob/54d6691f30fa0dc6849763da7ba78d5e4d236a42/app/helpers/form_helper.rb#L3) in the Ruby app
2. They're passed into the javascript application on the [document capture page](https://github.com/18F/identity-idp/blob/3fb9a2c099cd495b87d6e13299447b59006398bf/app/views/idv/shared/_document_capture.html.erb#L35)
3. They're loaded into the InPerson context, per this PR
4. They're passed down into the components in the PO search step, per this PR

As-is we'll have to redefine them in the identity-site repo

An alternative is that we could write a script to write to read from the Ruby file and write to a javascript file, and then publish the generated javascript file as part of an npm package. And then in the Ruby file add a file comment saying something like,
> If you change this set of states/territories be sure to go run script `bin/abc.sh` and publish a new version of the npm package

I'm going to go ahead and merge this PR. If we do refactor it in some way I think it should be part of a different ticket. But let me know your thoughts about it